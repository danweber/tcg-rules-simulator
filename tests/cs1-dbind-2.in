VERSION:2
# Suspend 2 of your opponent's Monsters. Then, return 1 of your opponent's suspended Monster to its owner's hand
#?? When this Monster attacks, trash the top card of your opponent's security stack.
HAND1:CS1-DoubleBind
FIELD1:Green,DW1-Bird Yellow,CS1-02
FIELD2:Suspend1,DW1-Van Suspend2,DW1-Van Safe,DW1-Van
GAME:P1 T3 MAIN 9
DUMP:HAND

#EXIT

ECHO: -- Now use option. Tough to see what happened, other tests will need to be more fine-grained.
CMD:PLAY CS1-DoubleBind
STEP:AUTOANSWER:CAPTURE
DUMP:HAND

