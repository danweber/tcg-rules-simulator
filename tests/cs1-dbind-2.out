FIELD1:Green,DW1-Birder Yellow,CS1-Papershield
FIELD2:Suspend1,DW1-Vanillamon Suspend2,DW1-Vanillamon Safe,DW1-Vanillamon
HAND1:CS1-Doublebind
HAND2:
GAME:P1 T3 MAIN 9
ECHO: -- Now use option. Tough to see what happened, other tests will need to be more fine-grained.
CAPTURE:Target VanillaMon 3
CAPTURE:Target VanillaMon 4
CAPTURE:Target VanillaMon 5
CAPTURE:Target VanillaMon 3
CAPTURE:Target VanillaMon 4
FIELD1:Green,DW1-Birder Yellow,CS1-Papershield
FIELD2:Suspend2,DW1-Vanillamon,REST Safe,DW1-Vanillamon
HAND1:
HAND2:DW1-Vanillamon
GAME:P1 T3 MAIN 2
